### PR TITLE
122/legal document

### LINKED ISSUE #122: Legal Document
Create top banner pointing to Legal Document

### PR DESCRIPTION
**Pull request by W3stside**
Closes #122 

- [X] Created LegalBanner that folds out and in on click

### CODE REVIEW

**Comment by W3stside** [comment]
Question: Does this need network detection of any kind? e.g showing a different banner for mainnet etc

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr124--dexreact.review.gnosisdev.com

**Comment by fleupold** [comment]
> Question: Does this need network detection of any kind? e.g showing a different banner for mainnet etc

cc @c3rnst

File: src/const.ts [hunk 1]
```diff
@@ -14,3 +14,10 @@ export const ALLOWANCE_FOR_ENABLED_TOKEN = new BN(2).pow(new BN(128)) // 3402823
 // Model constants
 export const FEE_DENOMINATOR = 1000 // Fee is 1/fee_denominator i.e. 1/1000 = 0.1%
 export const BATCH_TIME = 300
+
+export const LEGALDOCUMENT = {
+  CONTACT_ADDRESS: '[INSERT ADDRESS]',
+  POC_URL: '/',
+  TEXT: '',
```
Comment (alfetopito) [inline src/const.ts]: `TEXT` is not used anywhere. Did you intent to put the actual text in here?
If not used, I'd suggest to just remove it.

Comment (W3stside) [reply src/const.ts]: :+1:

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr124--dexreact.review.gnosisdev.com

**Comment by W3stside** [comment]
merging now - if different banner required we can create a new PR

**Comment by c3rnst** [comment]
> > Question: Does this need network detection of any kind? e.g showing a different banner for mainnet etc
> 
> cc @c3rnst

yes, the whole terms and privacy policy will have to look completely different on mainnet. This banner can only be implemented on Rinkeby. For Mainnet, we need proper tickbox checking before interacting on the site.

**Pull request merged**